### PR TITLE
style: ensure every prompt in the CLI has a FinalMessage 

### PR DESCRIPTION
**Pull request by efekarakus**
We now have every prompt give a short final message after a response is entered making our UX consistent.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: internal/pkg/term/prompt/prompt.go [hunk 1]
```diff
@@ -292,6 +292,13 @@ func WithFinalMessage(msg string) PromptConfig {
 	}
 }
 
+// WithConfirmFinalMessage sets a short final message for to confirm the user's input.
+func WithConfirmFinalMessage() PromptConfig {
+	return func(p *prompt) {
+		p.FinalMessage = color.Emphasize("Sure?")
```
Comment (iamhopaul123) [inline internal/pkg/term/prompt/prompt.go:298]: What does this look like 🤔 Will it be a breaking change for script users or it is just pure UI changes?

Comment (efekarakus) [reply internal/pkg/term/prompt/prompt.go:298]: It's pure UI changes. So for example, we go from:
```
Are you sure you want to delete application demo? [? for help] (Y/n) 
Are you sure you want to delete application demo?  Y
```
to
```
Are you sure you want to delete application demo? [? for help] (Y/n) 
Sure?  Y
```

**Review by iamhopaul123 — APPROVED** [review]
LGTM :shipit:

**Review by huanjani — APPROVED** [review]
LGTM

**Pull request merged**